<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgLayer</class>
 <widget class="QDialog" name="CDlgLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>345</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="eName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>52</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>52</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbHSpeed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>63</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbVSpeed">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>63</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>164</y>
     <width>52</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>197</y>
     <width>52</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>horizon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>197</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>vertical</string>
   </property>
  </widget>
  <widget class="QLabel" name="sType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Layers scroll both horizontally and vertically at a fraction of the speed of the main layer.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cCustom">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom / user defined</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>eName</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>cbHSpeed</tabstop>
  <tabstop>cbVSpeed</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgLayer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgLayer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
